### PR TITLE
Fixes for installing fontconfig 2.11.1 on OS X

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: packages/package_freetype_2_5_2/tool_dependencies.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
                     <environment_variable name="C_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include/freetype2</environment_variable>
                     <environment_variable name="CPLUS_INCLUDE_PATH" action="prepend_to">$INSTALL_DIR/include/freetype2</environment_variable>
                     <environment_variable action="prepend_to" name="PKG_CONFIG_PATH">$INSTALL_DIR/lib/pkgconfig</environment_variable>
+                    <environment_variable action="set_to" name="FREETYPE_ROOT_DIR">$INSTALL_DIR</environment_variable>
```
Comment (bgruening) [inline packages/package_freetype_2_5_2/tool_dependencies.xml]: @gregvonkuster latest Galaxy version will set this ENV by default, so not need to include it anymore.

**Comment by bgruening** [comment]
:+1:

File: packages/package_freetype_2_5_2/tool_dependencies.xml (hunk 1 above)
Comment (gregvonkuster) [inline packages/package_freetype_2_5_2/tool_dependencies.xml]: Thanks - I didn't know that.  Is is always <PKGNAME>_ROOT_DIR?

Comment (bgruening) [inline packages/package_freetype_2_5_2/tool_dependencies.xml]: Jupp: https://github.com/galaxyproject/galaxy/blob/dev/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py#L996

Comment (martenson) [inline packages/package_freetype_2_5_2/tool_dependencies.xml]: I think you cannot ever assume the repo is being installed on the Galaxy that has it though.

**Pull request merged**